bootstrap/terraform/main.tf

```diff
@@ -97,7 +97,8 @@ module "eks_blueprints_kubernetes_addons" {
   # Crossplane community AWS Provider deployment
   #---------------------------------------------------------
   crossplane_aws_provider = {
-    enable = true
+    enable                = true
+    provider_config = "aws-provider-config"
```

Comment:
For some reason github web ui, didn't format as I wanted. 
run `terraform fmt` to fix

Reply:
Will do.